index.bs

```diff
@@ -1685,6 +1617,76 @@ Methods</h4>
 			<em>Return type:</em> {{AudioTimestamp}}
 		</div>
 
+	: <dfn>resume()</dfn>
+	::
+		Resumes the progression of the {{AudioContext}}'s
+		{{BaseAudioContext/currentTime}} when it has
+		been suspended.
+
+		<div algorithm="AudioContext::resume()">
+			<span class="synchronous">When resume is called,
+			execute these steps:</span>
+
+			1. Let <var>promise</var> be a new Promise.
+
+			2. If the <em>control thread state</em> flag on the
+				{{AudioContext}} is <code>closed</code> reject the
+				promise with {{InvalidStateError}}, abort these steps,
+				returning <var>promise</var>.
+
+			3. If the {{AudioContext}} is not <a>allowed to
+				start</a>, append <var>promise</var> to
+				<a>pendingResumePromises</a> and abort these steps, returning
+				<var>promise</var>.
+
+			4. Set the <em>control thread state</em> flag on the
+				{{AudioContext}} to <code>running</code>.
+
+			5. <a>Queue a control message</a> to resume the {{AudioContext}}.
```

Comment:
Not sure what to do here, but the link to Queue a control message talks adding a message to the queue of an AudioContext.  Maybe that section should be changed to BaseAudioContext? Or maybe this needs to be fixed in a completely separate PR.

Reply:
Please file an issue on this. Otherwise, lgtm except for a lot of duplicated text which I'm not really sure how to fix that.